### ISSUE TITLE
Evaluation on custom dataset

### ISSUE THREAD

**Issue by Yuan1221** [skyhehe123/AutoRF-pytorch#3]
Hi, 

May I know if there's a script to evaluate images on custom datasets?

I would also like to know if it can be evaulated on datasets without annotations (labels and calibration matrices). 

Thank you.

**Comment by skyhehe123** [comment]
@Yuan1221 You can use pre-trained  monocular detection model to create 3D bbox, while calibration is a must to build projection between box-constrained nerf and rendering image. I currently only train this model on KITTI and it can learn reconstruction prior from the data. You can test on custom dataset by inputing the ROI of vehicle and 3D BBox (also with calibration).  I believe more plausible results can be achieved if train this model on larger auto-driving dataset, like NuScenes (as the paper does).
